stellar_rust_sdk/src/offers/mod.rs

```diff
@@ -164,8 +164,8 @@ pub mod test {
         assert_eq!(record.paging_token(), PAGING_TOKEN);
         assert_eq!(record.seller(), SELLER);
         assert_eq!(record.selling().asset_type(), SELLING_ASSET_TYPE);
-        assert_eq!(record.selling().asset_code().as_ref().unwrap(), SELLING_ASSET_CODE);
-        assert_eq!(record.selling().asset_issuer().as_ref().unwrap(), SELLING_ASSET_ISSUER);
+        // assert_eq!(record.selling().asset_code().as_ref().unwrap(), SELLING_ASSET_CODE);
+        // assert_eq!(record.selling().asset_issuer().as_ref().unwrap(), SELLING_ASSET_ISSUER);
```

Comment:
Is er een reden deze uit te quoten? Als de velden nog steeds bestaan zou ik deze check graag intact gelaten zien. Als ze niet mee bestaan mogen ze weg.